app.py

```diff
@@ -55,9 +55,12 @@ def render_template_with_context(template):
     else:
         grant_token = None
 
+    hypothesis_api_url=os.environ.get('HYPOTHESIS_API_URL')
```

Comment:
If you decide to keep the new `HYPOTHESIS_API_URL` env var (rather than using the existing `HYPOTHESIS_SERVICE` one instead) then I think it's a _required_ env var (the client will crash at boot without it) so this should just be `hypothesis_api_url=os.environ['HYPOTHESIS_API_URL']`, which will crash in the terminal if there's no `HYPOTHESIS_API_URL`, same as the publisher test site does for its other required env vars.